src/main/target/MIDELICF3/target.h

```diff
@@ -22,6 +22,19 @@
 
 #define TARGET_BOARD_IDENTIFIER "MIF3"
 
+
```

Comment:
Not sure how much value there is in updating the MIDELICF3 target - to my knowledge only a handful of prototype boards were ever built.

Reply:
Yeah true, I was just adding to all the cc2500 targets.